.travis.yml

```diff
@@ -1,21 +1,23 @@
 sudo: required
 
 services:
-  - docker
+- docker
 
 language: java
 jdk: openjdk8
 
-before_install: "cp .maven.settings.xml $HOME/.m2/settings.xml"
+before_install:
+- cp .maven.settings.xml $HOME/.m2/settings.xml
+- mvn fmt:check
 
-script: "mvn fmt:check verify cobertura:cobertura"
+script: mvn cobertura:cobertura-integration-test
```

Comment:
Why remove fmt:check and verify?

Reply:
I've moved it to the before_install, check https://github.com/ONSdigital/rm-sample-service/pull/66 for the reason :)